cloudcredential/v1/aws_manual.deepcopy_test.go

```diff
@@ -0,0 +1,77 @@
+package v1
+
+import (
+	"testing"
+
+	"k8s.io/apimachinery/pkg/runtime"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
The goal of this repo is to have minimal set of dependencies, I'd prefer to use go built-in mechanism over fancy assert libraries here.